pkg/es/config/config.go

```diff
@@ -366,3 +376,23 @@ func loadToken(path string) (string, error) {
 	}
 	return strings.TrimRight(string(b), "\r\n"), nil
 }
+
+// LoadTagsFromFile loads tags from a file
+func LoadTagsFromFile(filePath string) ([]string, error) {
+	file, err := os.Open(filepath.Clean(filePath))
+	if err != nil {
+		return nil, err
+	}
+	/* #nosec G307 */
```

Comment:
could you add the gosec description for g307? `Deferring a method which returns an error`. Which brings the question: what should be done if the closing of the file failed? Log a message?